apps/web/components/getting-started/components/ButtonLogic.tsx

```diff
@@ -0,0 +1,14 @@
+import { showToast } from "@calcom/ui";
+
+export const onButtonClick = async (nextStep: () => void, setButtonDisabled: (disabled: boolean) => void) => {
```

Comment:
```suggestion
export const onNextstepLogic = async (nextStep: () => void, setButtonDisabled: (disabled: boolean) => void) => {
```

Maybe we can rename this so its clearer what it does

Reply:
Sure it will be good for future references.